development/dev.env

```diff
@@ -18,3 +18,7 @@ REDIS_PORT=6379
 # Uncomment REDIS_SSL if using SSL
 # REDIS_SSL=True
 SECRET_KEY=012345678901234567890123456789012345678901234567890123456789
+NAUTOBOT_DB_HOST=${DB_HOST}
+NAUTOBOT_NAME=${DB_NAME}
```

Comment:
Should this be `NAUTOBOT_DATABASE`? There's no `NAUTOBOT_NAME` environment setting that I'm aware of.

Looking more closely at this, `development/nautobot_config.py` is the odd one out here in that it's using `DB_*` environment variables instead of the `NAUTOBOT_*` environment variables that are used by the default settings.py, the test nautobot_config.py, and the nautobot_config.py.j2 template -- so rather than adding these additional variables to the dev environment as redundant with the `DB_*` variables defined earlier in this file, can we instead *replace* those variables and fix `development/nautobot_config.py` accordingly?

Reply:
You are right about the NAUTOBOT_NAME should be NAUTOBOT_DATABASE. I was trying to match the `nautobot-server init`, but I missed this one.

As far as the DB_*, those are also used by the Postgres container to initialize these databases. Hence the additional variables.

Reply:
Seconded for keeping the variable names consistent across the development, test, and template-generated configs (from `nautobot-server init`).

Reply:
Thanks for the information! I didn't realize those were hard-coded names for Postgres to look for. I think it might still be good to change `development/nautobot_config.py` to use `NAUTOBOT_*` variables, just to be self-consistent, but I'm fine if you want to leave that out of this particular PR.